app/controllers/sign_up/completions_controller.rb

```diff
@@ -7,21 +7,18 @@ class CompletionsController < ApplicationController
 
     def show
       @view_model = view_model
-
-      if user_fully_authenticated? && session[:sp].present?
-        analytics.track_event(
-          Analytics::USER_REGISTRATION_AGENCY_HANDOFF_PAGE_VISIT,
-          service_provider_attributes
+      if show_completions_page?
+        track_agency_handoff(
+          Analytics::USER_REGISTRATION_AGENCY_HANDOFF_PAGE_VISIT
         )
       else
         redirect_to new_user_session_url
```

Comment:
Hm, I know this is what was there before, but it was confusing to me for a bit and maybe we should change it to `root_url` or `account_url`? It works and I don't have a strong feeling so I'll leave up to you.